javascript/javascript2/week1/Find-the-shortest-word.js

```diff
@@ -0,0 +1,18 @@
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+//notThisFunctionName(danishWords); // returns 'ø'
+findShortestWord(danishWords);
+
```

Comment:
if I run:
```
findShortestWord(["ø", "bil", "plante", "kaffe", "bog", "planetarium"]);
```
your code does not work. What is wrong?

Reply:
ok Rasmus i will recitify